TransitionSmurf/source/state_smurf/transition/StateTransition.cpp

```diff
@@ -3,34 +3,38 @@
 #include <bringauto/logging/Logger.hpp>
 
 namespace state_smurf::transition {
+
+	constexpr bringauto::logging::LoggerId logId = {.id = "StateTransition"};
```

Comment:
Please do not define logger template instances in .cpp file which does not belong to it.

Maybe create a new .hpp file where all loggers will be instantiated for a given library?

Reply:
How are these StateTransition loggers connected? You use them in multiple places but with different symbol.

Reply:
moved all logger definitions to headers